fitbenchmarking/cost_func/tests/test_cost_func.py

```diff
@@ -363,28 +385,45 @@ def test_hes_res(self):
         H, _ = self.cost_function.hes_res(params=[5],
                                           x=self.x_val,
                                           y=self.y_val)
-
-        expected = np.array([[[-1, -15, -21],
-                              [-1, -15, -21]],
-                             [[-1, -15, -21],
-                              [-1, -15, -21]]])
+        print(H)
```

Comment:
Think this has been left in unintentionally? There's also another print statement in `test_hes_res` for the poisson cost function